examples/blink/blink.ino

```diff
@@ -0,0 +1,17 @@
+#include "Arduino.h"
```

Comment:
Such example is already provided by Arduino built-in examples. What does it add? 
https://github.com/arduino/arduino-examples/blob/main/examples/01.Basics/Blink/Blink.ino